src/main/java/net/dv8tion/jda/api/entities/WebhookClient.java

```diff
@@ -42,8 +47,24 @@
  * @see Webhook
  * @see net.dv8tion.jda.api.interactions.InteractionHook
  */
-public interface WebhookClient<T>
+public interface WebhookClient<T> extends ISnowflake
```

Comment:
Possible to get/send/edit/delete in a specific thread for this? Possibly `setThread` or `setThreadId` on the restaction?